src/inverseRobot.js

```diff
@@ -7,7 +7,33 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const arrayValues = Object.values(robot);
+
+  for (let i = 0; i <= arrayValues.length; i++) {
+    let count = 0;
+    const elementI = arrayValues[i];
+
+    for (let j = 0; j <= arrayValues.length; j++) {
+      if (elementI === arrayValues[j]) {
+        count++;
+      }
+    }
+
+    if (count === 2) {
+      return null;
+    }
+  }
+
+  for (let key in robot) {
+    const savedKey = key;
+    const savedValue = robot[key];
+
+    delete robot[key];
+    key = savedValue;
+    robot[key] = savedKey;
+  }
+
+  return robot;
```

Comment:
you can just create new object before creating loop, and in the loop you can check - if value already exist in your new object - return null, if not - create it.
In helps you avoid difficult algorithm and redundant nested loop

Reply:
Not resolved, if you have any questions - ask them in `fe_chat`